rlkit/core/rl_algorithm.py

```diff
@@ -189,31 +185,17 @@ def train(self):
                 if self.train_embedding_source == 'initial_pool':
                     # embeddings are computed using only the initial pool of data
                     # sample data from posterior to train RL algorithm
-                    self.collect_data_from_task_posterior(idx=idx,
-                                                          num_samples=self.num_steps_per_task,
-                                                          add_to_enc_buffer=False)
-                elif self.train_embedding_source == 'posterior_only':
-                    self.collect_data_from_task_posterior(idx=idx, num_samples=self.num_steps_per_task, eval_task=False,
-                                                          add_to_enc_buffer=True)
+                    self.collect_data(self.num_steps_per_task, self.resample_z_train, np.inf, self.embedding_batch_size, add_to_enc_buffer=False)
```

Comment:
I think it's also okay to just get rid of this option altogether. I don't see when someone would use it at this point.

Reply:
do you think the same is true of `initial_pool`?

Reply:
This comment was addressed towards the initial_pool option.

Reply:
see `9161b6b`